ddterm/shell/windowmatch.js

```diff
@@ -43,9 +46,15 @@ export const WindowMatchGeneric = GObject.registerClass({
             }
         });
 
-        Meta.get_window_actors(this.display).forEach(actor => {
-            this._watch_window(actor.meta_window);
-        });
+        if (shellVersion < 48) {
```

Comment:
Please check if method exists instead of doing version check. It'll be both shorter and simpler.